src/structures/Invite.js

```diff
@@ -55,6 +55,12 @@ class Invite {
     this.code = data.code;
 
     /**
+     * The approximate number of online members of the guild this invite is for
+     * @type {number}
+     */
+    this.online = data.approximate_presence_count;
```

Comment:
this is a very vague and unintuitive property name, maybe just camel case the data name instead?